src/OpenLoco/src/World/CompanyAi/CompanyAiPathfinding.cpp

```diff
@@ -339,7 +338,7 @@ namespace OpenLoco::CompanyAi
     // company : _unk112C390
     //
     // return : _queryTrackRoadPlacementFlags, _queryTrackRoadPlacementMinScore, _queryTrackRoadPlacementMinWeighting
-    static void queryTrackPlacementScoreRecurse(Company& company, const World::Pos3 pos, const uint16_t tad, const bool unkFlag, QueryTrackRoadPlacementResult& totalResult, QueryTrackRoadPlacementState& state)
+    static void queryTrackPlacementScoreRecurse(Company& company, const World::Pos3 pos, const uint16_t tad, const bool unkFlag, const sfl::static_vector<uint8_t, 64>& validTrackIds, QueryTrackRoadPlacementResult& totalResult, QueryTrackRoadPlacementState& state)
```

Comment:
This `const sfl::static_vector<uint8_t, 64>& validTrackIds` is used in quite a few signatures in this unit now, so I feel like it warrants its own type.

Reply:
okay done.